packages/fxa-admin-panel/src/components/AccountSearch/index.tsx

```diff
@@ -126,6 +127,7 @@ export const AccountSearch = () => {
 
   const handleSubmit = (event: React.FormEvent) => {
     event.preventDefault();
+    const trimmedSearchValue = searchInput.trim();
```

Comment:
Is this from a previous PR or bad merge? Seems trimmedSearchValue is unused?

Reply:
Yeah it was -- removed now!